apps/studio-client/src/main/joo/com/coremedia/labs/translation/gcc/studio/GccProcessDefinitions.properties

```diff
@@ -74,7 +74,7 @@ ErrorCode_GCC_WF_40001_text=The GlobalLink configuration settings contain an uns
 ErrorCode_GCC_WF_40050_text=Encountered an illegal submission ID.
 ErrorCode_GCC_WF_50050_text=An xliff export failure occurred.
 ErrorCode_GCC_WF_61001_text=Failure while trying to cancel GlobalLink submission.
-Action_Tooltip_Cancel_Process=Cancel the selected process
+Action_Tooltip_Cancel_Process=Cancel the selected translation with GlobalLink
```

Comment:
We need a plural text here for multi-selection (and accordingly for de properties).